webpack.config.js

```diff
@@ -10,6 +11,13 @@ module.exports = {
 		),
 		new DependencyExtractionWebpackPlugin(),
 	],
+	resolve: {
+		extensions: [ '.json', '.js', '.jsx' ],
+		modules: [ path.join( __dirname, 'client' ), 'node_modules' ],
+		alias: {
+			wcstripe: path.resolve( __dirname, 'client' ),
```

Comment:
Added an alias for WP so that we can include with `import * from 'wcstripe/*` in our JS files

Reply:
🙇